team/mantras.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Team mantras
+description: A set of high level mantras the web and design team embrace
+---
+
+## Never remove data or functionality because of screen size
```

Comment:
For consistency, would it be better for each mantra to be communicated with the same force? As in, if others say "Do not", shouldn't we say "Do not" (or "Don't") here too? Never seems a bit too insistent.

Reply:
I don't think that this is always necessarily the best solution - I know this is going to escalate in a big debate, however I think you should aim to provide the correct amount of information/functionality for the context the user is in. More information/functionality does not always equal a better experience.

Reply:
Fair enough - I suppose I was saying I thought "never" was too strong wording in this case, but I'm open to being persuaded.

Especially, I think the "Never hide" in the next sentence is too strong - because as you suggest, one way to preserve functionality is to hide it behind a user action - click to expand, or a menu or something.

But I take your points. I think they're good.

@lilyvidenova if we expand on the point a bit to clarify that it's okay to *hide* functionality, as long as it's findable, do you still disagree? The point being that nothing you can do or can read on a desktop should be impossible to do / find on a phone?

I'd be interested to hear specific examples where you think it might be justified to remove information or functionality from phone users.